.github/workflows/linux-ci.yml

```diff
@@ -96,21 +96,29 @@ jobs:
           cmake -B build -GNinja -DCMAKE_BUILD_TYPE=Debug -DMLN_WITH_CLANG_TIDY=ON -DCMAKE_CXX_COMPILER_LAUNCHER=ccache -DMLN_WITH_COVERAGE=ON ${{ env.renderer_flag_cmake }}
           cmake --build build --target mbgl-core mbgl-test-runner mbgl-render-test-runner mbgl-expression-test
 
-      # mbgl-render (used for size test)
+      # mbgl-render (used for size test) & mbgl-benchmark-runner
 
-      - name: Build mbgl-render
+      - name: Build mbgl-render & mbgl-benchmark-runner
         if: matrix.renderer == 'drawable'
         run: |
-          cmake -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -B build-size -GNinja -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_CXX_COMPILER_LAUNCHER=ccache -DMLN_DRAWABLE_RENDERER=ON
-          cmake --build build-size --target mbgl-render
+          cmake -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -B build-release -GNinja -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_CXX_COMPILER_LAUNCHER=ccache -DMLN_DRAWABLE_RENDERER=ON
```

Comment:
Why do we need `Debug` build above to run tests? Can we just build everything with `RelWithDebInfo`?

Reply:
I guess because you don't have optimisations, but `RelWithDebInfo` should have all the debug symbols. As Linux runners are not so critical, I guess it's fine for now, but just a tip how to potentially speed things up a bit.